core/constant/status.go

```diff
@@ -1,22 +1,24 @@
 package constant
 
 const (
-	StatusSuccess = "success"
-	StatusFailed  = "failed"
+	StatusSuccess = "Success"
+	StatusFailed  = "Failed"
 
 	// node
-	StatusWaiting     = "waiting"
-	StatusDownloading = "downloading"
-	StatusPacking     = "packing"
-	StatusSending     = "sending"
-	StatusStarting    = "starting"
-	StatusHealthy     = "healthy"
-	StatusUnhealthy   = "unhealthy"
-	StatusUpgrading   = "upgrading"
-	StatusRunning     = "running"
-	StatusFree        = "free"
-	StatusBound       = "bound"
-	StatusExceptional = "exceptional"
-	StatusRetrying    = "retrying"
-	StatusLost        = "lost"
+	StatusWaiting     = "Waiting"
+	StatusPacking     = "Packing"
+	StatusSending     = "Sending"
+	StatusStarting    = "Starting"
+	StatusHealthy     = "Healthy"
+	StatusUnhealthy   = "Unhealthy"
+	StatusUpgrading   = "Upgrading"
+	StatusRunning     = "Running"
+	StatusFree        = "Free"
+	StatusBound       = "Bound"
+	StatusExceptional = "Exceptional"
+	StatusRetrying    = "Retrying"
+	StatusLost        = "Lost"
+
+	StatusEnable  = "Enable"
+	StatusDisable = "Disable"
 )
```

Comment:
In terms of regular adjustments needed to be made in this code snippet, no significant improvements were necessary for clarity. Please ensure you have the latest version of the constant library before reapplying to see if there's anything new added.

Since we're discussing recent changes, here are a few observations:

1. No syntax errors found.
2. A slight error occurred while converting status strings from `Status` enum into their corresponding Go string equivalents: StatusFailed should become Failed instead.
3. Typically uppercase "Error" is used for an exceptional case, not "Exceptional", but no such inconsistency was observed within this script (assuming it only applies globally, like with StatusEnabled).

With these points being minor and insignificant, no further actions need immediate modification as long as you don't plan on adding more complexities or using specific statuses beyond the available ones. If you want to make major adjustments based on this observation, consider updating everything with uppercase Exceptional rather than Exceptional.

As far as optimization goes, given that all values provided are predefined constants and do not require explicit conversion from int or float types, there isn't much room for improvement without altering the structure of how status is communicated between components/other parts of the process.

For instance, instead of hard-coding 'waiting' status directly after its definition, you can use the value itself:
`// node ```
-	StatusWaiting     := "waiting"
```